src/completion_queue.rs

```diff
@@ -25,13 +26,13 @@ pub struct CompletionQueue {
 
 impl CompletionQueue {
     /// Get the internal cq ptr
-    pub(crate) const fn as_ptr(&self) -> *mut ibv_cq {
```

Comment:
CQ is not transparent for user, so pub(crate) is not necessary.

Reply:
Specific is better than implicit. So I recommend to keep `pub(crate)`.

Reply:
https://github.com/datenlord/async-rdma/pull/28/commits/61644b432d838cfa16feb63b2e17b222d4125b17 fixed

Reply:
https://github.com/datenlord/async-rdma/pull/28/commits/e9f2f1b34afc6a9bfc4d0e0a8441a42890c80b36 fixed